giddy/markov.py

```diff
@@ -1804,27 +1908,28 @@ def __init__(self, y):
         r_asc = np.array([rankdata(col, method='ordinal') for col in y.T]).T
         # ranks by high (1) to low (n)
         self.ranks = r_asc.shape[0] - r_asc + 1
-        frm = Markov(self.ranks)
-        self.p = frm.p
-        self.transitions = frm.transitions
-
-    @property
-    def steady_state(self):
-        if not hasattr(self, '_steady_state'):
-            self._steady_state = STEADY_STATE(self.p)
-        return self._steady_state
-
-    @property
-    def fmpt(self):
-        if not hasattr(self, '_fmpt'):
-            self._fmpt = fmpt(self.p)
-        return self._fmpt
-
-    @property
-    def sojourn_time(self):
-        if not hasattr(self, '_st'):
-            self._st = sojourn_time(self.p)
-        return self._st
+        super(FullRank_Markov, self).__init__(self.ranks)
+        # frm = Markov(self.ranks)
+        # self.p = frm.p
+        # self.transitions = frm.transitions
+
+    # @property
```

Comment:
delete this if it's not useful... don't comment?

Reply:
Will do!